docs/design/ovs-pipeline.md

```diff
@@ -929,12 +1135,53 @@ The first flow outputs all unicast packets to the correct port (the port was
 resolved by the "dmac" table, [L2ForwardingCalcTable]). IP packets for which
 [L2ForwardingCalcTable] did not set bit 16 of NXM_NX_REG0 will be dropped.
 
+## Tables (AntreaProxy is disabled)
+
+![OVS pipeline](../assets/ovs-pipeline.svg)
+
+### DNATTable (40)
+
+This table is create when feature AntreaProxy is disabled. Its only job is to
+send traffic destined to Services through the local gateway, without any
```

Comment:
gateway -> gateway interface, to be clearer.

Reply:
Thanks, updated.